opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -199,67 +223,62 @@ message HistogramValue {
     // Exemplars are example points that may be used to annotate aggregated
     // Histogram values. They are metadata that gives information about a
     // particular value added to a Histogram bucket.
-    message Exemplar {
-      // Value of the exemplar point. It determines which bucket the exemplar
-      // belongs to.
-      double value = 1;
 
-      // The observation (sampling) time of the above value.
-      google.protobuf.Timestamp timestamp = 2;
+    // Value of the exemplar point. It determines which bucket the exemplar belongs to.
+    double exemplar_value = 2;
 
-      // Contextual information about the example value.
-      map<string, string> attachments = 3;
-    }
+    // The observation (sampling) time of the above value.
+    int64 exemplar_timestamp_unixnano = 3;
 
-    // Exemplars are example points that may be used to annotate aggregated
-    // Histogram values.
-    Exemplar exemplar = 2;
```

Comment:
I am not sure if this is worth it. Exemplars will not be used for all the histograms, and the current change increases the memory usage for all the histograms to avoid one extra allocation. Can we split this in a separate PR and discuss?